[Ready for Review] add case studies chapter for RR (#1248)
alan-turing-institute/the-turing-way: book/website/reproducible-research/case-studies.md

(rr-case-studies)=
# Case Studies: Reproducible Research

## Prerequisites / recommended skill level

> This chapter assumes familiarity with all topics covered in the [chapter: guide to reproducible research]. 

Reviewer: ```suggestion
This chapter assumes familiarity with several topics covered in this [Guide to Reproducible Research]. 
```

Author: agreed

> The actually required skills depend on the case study since not all examples
> make use of all methods covered in the guide to reproducible research.

Reviewer: ```suggestion
Since not all examples will demonstrate the use of the same combinations of methods covered in this guide, the prerequisite and skill level will vary across the case studies.
Therefore, we recommend the authors to indicate those skills for each case study separately.
```

## Summary

> In this chapter a selection of case studies is presented that can be used to 
> see how reproducible research methods can be used in practice. 
> If you think that you might have an exampel to contribute yourself, feel free
> to open an issue at [issues].

Reviewer: ```suggestion
In this chapter, a selection of case studies is presented that can be used to 
see how reproducible research methods are used in practice and applied across fields. 
If you think that you might have an example to contribute yourself, feel free
to open an issue at [issues].
```

## How this will help you/ why this is useful

Learning by example can be an effective way to learn new concepts. 

> Maybe you can start your personal Turing Way by adapting one of the examples 

Reviewer: ```suggestion
Maybe you can start your personal project by adapting one of the examples 
```
